src/sites/image/fotokiz.com.js

```diff
@@ -1,32 +0,0 @@
-_.register({
-  rule: {
-    host: [
-      /^crownimg\.com$/,
-      /^fotokiz\.com$/,
-      /^imgbaron\.com$/,
-      /^imgsen\.com$/,
-      /^imgsto\.com$/,
-      /^kropic\.com$/,
-      /^kvador\.com$/,
-      /^picbaron\.com$/,
-      /^picdollar\.com$/,
-      /^pics4upload\.com$/,
-      /^silverpic\.com$/,
-      /^barbit\.net$/,
-      /^pics4you\.net$/,
-      /^imgstar\.eu$/,
-      /^www\.fappic\.com$/,
-    ],
-  },
-  async ready () {
-    const i = $.$('img.pic');
-    if (i) {
-      // second stage
```

Comment:
as you are removing sites not longer exist, the rest of the code should be untouched.
but we are missing the "return;" line.